src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/LanguageVersionExtensions.cs

```diff
@@ -2,13 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 namespace Microsoft.CodeAnalysis.CSharp.Shared.Extensions
 {
     internal static class LanguageVersionExtensions
     {
         public static bool IsCSharp9OrAbove(this LanguageVersion languageVersion)
             => languageVersion >= LanguageVersion.CSharp9;
+
+        public static bool IsCSharp10OrAbove(this LanguageVersion languageVersion)
+            => languageVersion >= LanguageVersion.Preview;
```

Comment:
Curious how would you ensure this to not be forgotten when C# 10 is released.

Reply:
We did the same thing for the C# 9 method right above this. Thankfully no one removed the method because that's how I found where to add this one. 😄